.changeset/neat-impalas-drive.md

```diff
@@ -0,0 +1,6 @@
+---
+'@keystonejs/adapter-knex': major
+'@keystonejs/adapter-mongoose': major
```

Comment:
Do these packages need bumps?

Reply:
Yes because they depend on `@keystonejs/fields` for the default id fields so the GraphQL schema they generate has changed. (they wouldn't if we didn't want this to be a major but because we want to, these need to have bumps)